extensions/kogito/deployment/src/test/java/io/quarkus/kogito/drools/Person.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Please remove this copyright header.... you get the idea.

Reply:
@dmlloyd is it our official position? I remember there was some disagreement about it so for now, I was conservative about it.

Reply:
It is official AFAIK

Reply:
I removed all headers.